lab-max/server.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const http = require('http');
+const url = require('url');
+const queryString = require('querystring');
+
+const cowsay = require('cowsay');
+
+const parseBody = require('./lib/parse-body');
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer(function(req, res){
+  req.url = url.parse(req.url);
+  req.url.query = queryString.parse(req.url.query);
+
+  if (req.method === 'GET' && req.url.pathname === '/'){
+    res.writeHead('200', {'Content-Type': 'text/plain'});
+    res.write('hello world\n');
+    res.end();
+  }
+
+  if (req.method === 'GET' && req.url.pathname === '/cowsay'){
+    if (req.url.query.text){
+      res.writeHead('200', {'Content-Type': 'text/plain'});
```

Comment:
@meeksfred as odd is it may seem initially, best practice is to break up the way this is written into new lines - for example:

```
res.writeHead(200, {
  'Content-Type' : 'text/plain'
});
```